docs/source/ext/github_tools.py

```diff
@@ -19,7 +19,7 @@
 from datetime import datetime, timedelta
 from subprocess import check_output
 
-from urllib.request import urlopen, Request
```

Comment:
Could you explain why you remove `urlopen`?  The link that you provide does not do it and I do not understand your change.

Reply:
As I mention it's a vulnerable calling function that can lead to open up the file too instead of URL only.